src/lib/encoder/Decoder.h

```diff
@@ -30,6 +30,7 @@ class Decoder
 
 protected:
 	unsigned _eccBytes;
+	unsigned _eccBlockSize;
```

Comment:
I'm not sure if this actually *needs* to be a member variable. But it's somewhat moot I think.